libs/src/mail/gmail/label/candidate.go

```diff
@@ -90,8 +92,15 @@ var (
 			TextColor:       "#ffffff",
 		},
 	}
-	// TODO
 	// JobsVerified is the @SRC/Jobs/Verified label for verified job opportunities.
-	// BackgroundColor is #2da2bb
```

Comment:
Use this background color?

Reply:
Just updated, figured the black would make it look more "official" and differentiate from the green / blue of Interested./ Opportunity